src/desktop/apps/feature/server.tsx

```diff
@@ -40,7 +40,7 @@ app.get(
         },
         locals: {
           ...res.locals,
-          assetPackage: "feature-reaction",
+          assetPackage: "feature",
```

Comment:
I _believe_ this `assets/feature.tsx` file can be deleted as well as `apps/feature-reaction`. We're not worrying about the old apps anymore right? In which case there's no real server-side functionality here and all we need to do is plug the reaction routes into `v2/Apps/getAppRoutes`.

Reply:
See here: https://github.com/artsy/force/blob/master/src/v2/Apps/getAppRoutes.tsx